cli/src/cli_util.rs

```diff
@@ -3387,6 +3387,7 @@ impl CliRunner {
     #[must_use]
     #[instrument(skip(self))]
     pub fn run(mut self) -> ExitCode {
+        clap_complete::CompleteEnv::with_factory(crate::commands::default_app).complete();
```

Comment:
nit: `.try_complete()` and return `ExitCode`? There might be custom resource cleanup after `run()`.